modules/apps/object/object-api/src/main/java/com/liferay/object/field/business/type/ObjectFieldBusinessType.java

```diff
@@ -47,7 +47,7 @@ public default Set<String> getAllowedObjectFieldSettingsNames() {
 
 	public String getDDMFormFieldTypeName();
 
-	public default String getDDMFormFieldTypeName(boolean localized) {
+	public default String getDDMFormFieldTypeName(ObjectField objectField) {
```

Comment:
Do we still need this change? I think we can keep the current code with this new solution.